### PR TITLE
Support blocks

### PR DESCRIPTION
**Pull request by schwers**

### CODE REVIEW

**Comment by schwers** [comment]
:eyeglasses: @ajacksified

**Comment by ajacksified** [comment]
:fish:

File: models/BlockedUser.es6.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import Base from './base';
+
+export default class BlockedUser extends Base {
+  _type = 'BlockedUser';
+
+
```
Comment (ajacksified) [inline models/BlockedUser.es6.js]: extra newline 😱

**Comment by ajacksified** [comment]
merged

**Pull request closed without merging**